assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -255,6 +255,22 @@ describe( 'modules/analytics properties', () => {
 				expect( console ).toHaveErrored();
 			} );
 		} );
+		describe( 'getPropertiesIncludingGA4', () => {
+			it( 'returns a sorted list of ua and ga4 properties ', async () => {
+				const testAccountID = fixtures.profiles[ 0 ].accountId; // eslint-disable-line sitekit/acronym-case
+				const accountID = testAccountID;
+
+				// Load data into stores
+				registry.dispatch( STORE_NAME ).receiveGetProperties( fixtures.propertiesProfiles.properties, { accountID } );
+				registry.dispatch( 'modules/analytics-4' ).receiveGetProperties( fixtures.propertiesGA4, { accountID } );
+
+				const properties = registry.select( STORE_NAME ).getPropertiesIncludingGA4( testAccountID );
+
+				expect( properties ).toMatchSnapshot();
+				expect( properties ).toHaveLength( 23 );
+			} );
```

Comment:
I would also like to test for when the `modules/analytics-4` datastore is not available, but I don't any way to conditionally load data stores when testing

the test util registerAllStoresOn registers all the stores!
https://github.com/google/site-kit-wp/blob/4cac32bb3a3a5fc977d07546ef3fe5d7f3bc94c3/tests/js/utils.js#L359